test/UtilsTest.php

```diff
@@ -254,7 +254,9 @@ public function testCheckModuleInstallation(){
 				]],
 			]);
 		
-		$this->assertTrue(Utils::CheckModuleInstallation('itop-structure', true, $oRestClient));
+		$this->assertTrue(Utils::CheckModuleInstallation('itop-structure', false, $oRestClient));
+		$this->assertTrue(Utils::CheckModuleInstallation('itop-structure/0.0.0', false, $oRestClient));
+		$this->assertFalse(Utils::CheckModuleInstallation('itop-structure/1.2.3', false, $oRestClient));
```

Comment:
I'm not sure to understand this test case, `itop-structure` was introduced in 3.0 with first version being `itop-structure/3.0.0`, so how can the test for "0.0.0" return true, but not the test for "1.2.3"?